OpenKh.Patcher/PatcherProcessor.cs

```diff
@@ -279,5 +284,79 @@ private static void PatchAreaDataScript(Context context, List<AssetFile> sources
 
             Kh2.Ard.AreaDataScript.Write(stream.SetPosition(0), scripts.Values);
         }
+
+        private static void PatchListReplace(Context context, List<AssetFile> sources, Stream stream)
+        {
+            foreach (var source in sources)
+            {
+                var deserializer = new DeserializerBuilder().IgnoreUnmatchedProperties().Build();
+                Dictionary<string, byte> characterMap = new Dictionary<string, byte>() {
+                            { "Sora", 1 }, { "Donald", 2 }, { "Goofy", 3 },  { "Mickey", 4 },  { "Auron", 5 }, { "PingMulan",6 }, { "Aladdin", 7 },  { "Sparrow", 8 }, { "Beast", 9 },  { "Jack", 10 },  { "Simba", 11 }, { "Tron", 12 }, { "Riku", 13 }, { "Roxas", 14}
+                        };
+                    string sourceText = File.ReadAllText(context.GetSourceModAssetPath(source.Name));
+                    if (source.Name.Contains("enc"))
+                    {
+                        byte[] data = System.Convert.FromBase64String(sourceText);
+                        sourceText = System.Text.ASCIIEncoding.ASCII.GetString(data);
+                    }
+                    switch (source.Type)
+                    {
+                        case "trsr":
+                            var TrsrList = Kh2.SystemData.Trsr.Read(stream).ToDictionary(x => x.Id, x => x);
```

Comment:
Please use camel-case. Your `TrsrList` would become `trsrList`. This is to keep consistency with the rest of the code base.